src/blocks/DetailsBlock/DetailsBlock.fragment.graphql

```diff
@@ -0,0 +1,4 @@
+fragment DetailsBlock on DetailsBlockRecord {
```

Comment:
Please make sure to also include the migration for this block in the PR. We might be blocked by #164 by this 🤷 .